components/fieldgroups/experience-event/industry-verticals/experienceevent-live-event-ticket-purchase.json

```diff
@@ -0,0 +1,184 @@
+              "description": "Ticket(s) Purchased",
+              "title": "Tickets",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "xdm:barcodeID": {
+                    "title": "Ticket Bar Code",
+                    "description": "Unique bar code assigned to a ticket.",
+                    "type": "string",
+                    "meta:xdmType": "string"
+                  },
+                  "xdm:deliveryMethod": {
+                    "title": "Delivery Method",
+                    "description": "Ticket Delivery Method, for example, postal or digital.",
+                    "type": "string",
+                    "meta:xdmType": "string",
+                    "meta:enum": {
+                      "postal": "Postal Delivery",
+                      "digital": "Digital Delivery"
+                    }
+                  },
+                  "xdm:rowName": {
+                    "title": "Seat Row Name",
+                    "description": "Seat row name as printed on the ticket",
+                    "type": "string",
+                    "meta:xdmType": "string"
+                  },
+                  "xdm:seatingArea": {
+                    "title": "Seating Area",
+                    "description": "Seating area as printed on ticket",
+                    "type": "string",
+                    "meta:xdmType": "string"
+                  },
+                  "xdm:seatName": {
+                    "title": "Seat Name",
+                    "description": "Seat name as printed on ticket.",
+                    "type": "string",
+                    "meta:xdmType": "string"
+                  },
+                  "xdm:seatingAreaNumeric": {
+                    "title": "Seating Area Numeric",
+                    "description": "Numeric Seating Area value",
+                    "type": "integer",
+                    "meta:xdmType": "int"
+                  },
+                  "xdm:sectionCategory": {
+                    "title": "Seating Section Category",
+                    "description": "Category of section, used to describe tier of ticket, balcony, box, etc...",
+                    "type": "string",
+                    "meta:xdmType": "string"
+                  },
+                  "xdm:sectionGroup": {
+                    "title": "Seating Section Group",
+                    "description": "Section Group as printed on Ticket",
+                    "type": "string",
+                    "meta:xdmType": "string"
+                  },
+                  "xdm:sectionLevel": {
+                    "title": "Seating Section Level",
+                    "description": "Section Level as printed on ticket.",
+                    "type": "string",
+                    "meta:xdmType": "string"
+                  },
+                  "xdm:sectionName": {
+                    "title": "Seating Section Name",
+                    "description": "",
```

Comment:
Please add the descriptions for all the properties

Reply:
added